### ISSUE TITLE
The PHP-8.0.0 does not have the version info

### ISSUE THREAD

**Issue by markwu** [phpbrew/phpbrew#1211]
### Output

module folder
![image](https://user-images.githubusercontent.com/196861/100718400-4060f700-33f6-11eb-8d27-0da9200dc84e.png)

httpd.conf
![image](https://user-images.githubusercontent.com/196861/100718948-edd40a80-33f6-11eb-9701-df86a73dfd92.png)

### Expected Result

The binary should named with version `libphp8.0.0.so`

### Command

```
phpbrew --debug install --jobs "$(nproc)" 8.0.0 +default +dbs +dba +apxs2 +ftp +soap +gettext +exif +iconv +intl +bz2 +zlib
```

### Platform

OS:

    macOS 10.15.7

Running PHP:  

    PHP-7.4.12

Installing PHP:

    PHP-8.0.0

**Comment by mahagr** [comment]
```
===> Checking patches...
Checking patch for replace apache php module name with custom version name
0 changes patched.
```
Likely because of there's no patch for that.

**Comment by peter279k** [comment]
I think it can rename the `libphp.so` to `libphp8.0.0.so` manually and it can make them identified with manual way.

**Comment by markwu** [comment]
It better managed by phpbrew as it used to be.

**Comment by peter279k** [comment]
You're right. It seems that the [patch parser for Apache2 module](https://github.com/phpbrew/phpbrew/blob/master/src/PhpBrew/Patches/Apache2ModuleNamePatch.php) should be updated for parsing `PHP 8.x` patches.

**Comment by peter279k** [comment]
After digging the `configure` file that generated by `buildconf` from PHP source, the PHP-7.4 configure file is different from PHP-8.

Here the steps I use to notice above situation:

- Prepare the Ubuntu 20.04 or other distributions that can run the `autoconf` command.
- Install the `autoconf` command. Running the `sudo apt-get install autoconf` in Ubuntu 20.04.
- Running the `cd ~/` to change directory to the user home directory.
- Download the PHP-7.4 source archive file with `wget https://www.php.net/distributions/php-7.4.29.tar.gz`.
- Running the `tar -xzf php-7.4.29.tar.gz` to extract the archived gzip file.
- Download the PHP-8.0.19 source archive file with `wget https://www.php.net/distributions/php-8.0.19.tar.gz`.
- Running the `tar -xzf php-8.0.19.tar.gz` to extract the archived gzip file.
- Running the `cd php-7.4.29` then running the `./buildconf --force` to regenerate the `configure` file.
- Running the `cd php-8.0.19` then running the `./buildconf --force` to regenerate the `configure` file.

We can see that related `libphp.so` variable defined in the line `6270`, `6271` and `6272` of `~/php-7.4.29/configure`:

```
......
 SAPI_SHARED=libs/libphp$PHP_MAJOR_VERSION.$SHLIB_DL_SUFFIX_NAME
 SAPI_STATIC=libs/libphp$PHP_MAJOR_VERSION.a
 SAPI_LIBTOOL=libphp$PHP_MAJOR_VERSION.la
......
```

We can see that related `libphp.so` variable defined in the line `6256`, `6257` and `6258` of `~/php-8.0.19/configure`:

```
.....
SAPI_SHARED=libs/libphp.$SHLIB_DL_SUFFIX_NAME
SAPI_STATIC=libs/libphp.a
SAPI_LIBTOOL=libphp.la
......
```

I also see that the `PHP_MAJOR_VERSION` pattern is not found in `~/php-8.0.19/build/Makefile.global` file.

That's the reason why the [patch parser for Apache2](https://github.com/phpbrew/phpbrew/blob/master/src/PhpBrew/Patches/Apache2ModuleNamePatch.php) will not find any version info for the `PHP-8.x` because the file contents are changed.

@markwu, do have find any version info about `libphp.so` in `PHP-8.x` version?

**Comment by peter279k** [comment]
I've created the PR #1265 to fix this issue :).

**Comment by peter279k** [comment]
This issue is resolved by PR #1265.
